README.md

```diff
@@ -16,49 +16,14 @@
 </p>
 <br />
 
+> Note: We now recommend using our Helm chart to deploy Sombra: https://github.com/transcend-io/helm-charts. This module still works in many cases, but may not always be fully up-to-date.
```

Comment:
If interested, an [Alert](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) might be a nice way to make this stand out when viewed on GitHub:

> [!NOTE] 
> We now recommend using our Helm chart to deploy Sombra: https://github.com/transcend-io/helm-charts. This module still works in many cases, but may not always be fully up-to-date.

```suggestion
> [!NOTE]
> We now recommend using our Helm chart to deploy Sombra: https://github.com/transcend-io/helm-charts. This module still works in many cases, but may not always be fully up-to-date.
```